core/templates/dev/head/domain/exploration/ExplorationObjectFactory.js

```diff
@@ -100,6 +100,10 @@ oppia.factory('ExplorationObjectFactory', [
           INTERACTION_DISPLAY_MODE_INLINE);
     };
 
+    Exploration.prototype.getStates = function() {
```

Comment:
I feel pretty bad about exposing this, but wasn't sure what better way there was?

Reply:
Any reason you can't use explorationStatesService, like ComputeGraphService does?

If there's a good reason, then I think exposing is OK.

Reply:
Do you mean graphDataService? It doesn't look like ComputeGraphService uses that.

explorationStatesService seems like it's pretty bound to editor functionality exclusively, so I don't think so

Reply:
Ah yes, you're right. OK, I'm fine with exposing it.